src/ui-components/input-field/input-field.spec.js

```diff
@@ -0,0 +1,315 @@
+import React from "react";
+import { shallow } from "enzyme";
+import InputField from "./input-field.js";
+import { CSS_CLASSES } from "./constants.js";
+
+describe("Input field", () => {
+  
+  it("mounts component", () => {
+    const wrapper = shallow(<InputField />);
+    expect(wrapper).toBeTruthy();
+  });
+
+  it("is not focused", () => {
+    const wrapper = shallow(<InputField />);
+    expect(wrapper.state().isInputFieldFocused).toEqual(false);
+  })
+
+  describe("props are set correctly", () => {
+
+    function testInputFieldProps(givenData, expectedResults) {

```

Comment:
We could destructure `givenData` and `expectedResults` params here like: `const { id, type, required, value } = givenData` so we could, later on, access them directly.

-  [ ] Use destructuring for easier param access.

Reply:
Since we don't need to reuse params too often, in this case, it would require more code and it is  personal preference - I am refuting it.